tasks/task183_rhyme_generation.json

```diff
@@ -0,0 +1,6032 @@
+{
+    "Contributors": [
+        "Ethan Kim"
+    ],
+    "Source": [
+        "Synthetic"
+    ],
+    "Categories": [
+        "Text Generation"
+    ],
+    "Definition": "Given an input word generate a list of words that rhyme exactly with that word",
```

Comment:
Some suggestions: 
 - Say that, if no word that rhymes found, output "No ...". 
 - One current issue is whether we have a good way of evaluating the output (like, what if instead of "pooh queue poo koo du" one generates "poo koo du pooh queue"?) To address these concerns, I suggest: 
       - You change the output to a list. For example, 
```
"output": [
    "pooh queue poo koo du"
]
```
could be: 
```
"output": [
    "pooh", "queue", "poo", "koo", "du"
]
```
      - Change the task description such that, it asks for one word, rather than a list of words (which can be long and open-ended).